recipes/tophu/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: tophu
+  version: {{ version }}
+
+source:
+  url: https://github.com/isce-framework/tophu/archive/v{{ version }}.tar.gz
+  sha256: 691b0cae0bde670a4b430bcb1a6d2ec030e99f34c45e655e50c82c102bcda0d1
+
+build:
+  script: {{ PYTHON }} -m pip install --no-deps . -vv
+  number: 0
+  skip: true  # [win or osx or py<38]
```

Comment:
Why not noarch? If it is b/c of missing dependencies we still prefer to make it noarch to avoid CI waste and let the solver deal with the missing dependencies.

Reply:
Unfortunately, the package is broken on macOS right now due to some issues with an upstream dependency that I haven't sorted out yet. It and its dependencies will install on macOS, but then will encounter errors at runtime. So the package could become noarch in the future if/when this issue is resolved, but for now I'd prefer to make it available on Linux only.

Reply:
You can still make it "some-arch" for Linux only. See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies

Reply:
TIL, thanks.

The examples in the docs you linked seem to indicate that that I should use the appropriate preprocessor selector along with the `__linux` virtual package, but the linter doesn't seem to like that change. Should I remove the selector?

Reply:
> the linter doesn't seem to like that change. Should I remove the selector?

That should be fixed by now but no, don't remove it. Let's ignore the linter for now.